spec/cancan/model_adapters/active_record_adapter_spec.rb

```diff
@@ -237,101 +237,84 @@ class User < ActiveRecord::Base
     end
 
     it 'has false conditions if no abilities match' do
-      expect(@ability.model_adapter(Article, :read).conditions).to eq("'t'='f'")
+      expect(@ability.model_adapter(Article, :read).database_records.to_sql.strip.squeeze(' '))
```

Comment:
No, it shouldn't still be in place - the entire underlying mechanism for generating queries has changed, and in such a way that the previous placeholder 't=f' query isn't present any more.